core/platform/datastore/gae_datastore_services.py

```diff
@@ -47,16 +47,32 @@
 UserProperty = ndb.UserProperty
 
 
-def put_multi(models):
+def get_multi(keys):
+    """Fetches a sequence of keys.
+
+    Args:
+        keys: list(str). The keys to look up.
+
+    Returns:
+        list(ndb.Model). List whose items are either a Model instance or None if
+        the key wasn't found.
+    """
+    return ndb.get_multi(keys)
+
+
+def put_multi(models, update_last_updated_time=True):
     """Stores a sequence of Model instances.
 
     Args:
         models: datastore_services.Model. A sequence of Model instances.
+        update_last_updated_time: bool. Whether to update the last_updated field
+            of the entities.
 
     Returns:
         list(str). A list with the stored keys.
     """
-    return ndb.put_multi(models)
+    return ndb.put_multi(
```

Comment:
@vojtechjelinek FYI, we won't be able to pass arbitrary args to put_multi anymore. We'll need to find a different way to update these fields efficiently.

Reply:
@brianrodri I think this will need a longer discussion. Could you please start an email thread (or meeting) with @vojtechjelinek and me on this?

Also please add a TODO here and an issue describing exactly what is required (per our discussion).